docs/content/en/install/eks/_index.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Install K8ssandra on EKS"
+linkTitle: "Amazon EKS"
+weight: 2
+description: >
+  Complete **production** ready environment of K8ssandra on Amazon Elastic Kubernetes Service (EKS).
+---
+
+Amazon [Elastic Kubernetes Service](https://aws.amazon.com/eks/features/) or "EKS" is a managed Kubernetes service that makes it easy for you to run Kubernetes on AWS and on-premises. EKS is certified Kubernetes conformant, so existing applications that run on upstream Kubernetes are compatible with EKS. AWS automatically manages the availability and scalability of the Kubernetes control plane nodes responsible scheduling containers, managing the availability of applications, storing cluster data, and other key tasks.
+
+{{% alert title="Tip" color="success" %}}
+Also available in followup topics are post-install steps and role-based considerations for [developers]({{< relref "/quickstarts/developer">}}) or [site reliability engineers]({{< relref "/quickstarts/site-reliability-engineer">}}) (SREs).
```

Comment:
Do these post-install and consideration topics exist such that they could be linked to?

Reply:
I would comment this out until you have substantiating doc to link to... otherwise it is just frustrating to the user.